index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <form onsubmit="formSubmit(event)">
+        <label for="firstName">First Name:</label>
+        <input type="text" name="firstName" id="firstName" required>
+        <label for="lastName">Last Name:</label>
+        <input type="text" name="lastName" id="lastName" required>
+        <label  for="age">Age:</label>
+        <input onkeyup="validateInput('age')" type="number" name="age" id="age" required>
+        <label for="email">Email:</label>
+        <input type="email" name="email" id="email" required>
+        <label  for="phone">Phone Number:</label>
+        <input onkeyup="validateInput('phone')" type="tel" name="phone" id="phone" required>
+        <label>Gender</label>
+        <div class="gender-container">
+            <label for="male">Male</label>
+        <input type="radio" name="gender" id="male" required>
+        <label for="female">Female</label>
+        <input type="radio" name="gender" id="female" required>
+        </div>
+        <p id="error"></p>
+        <button type="submit">submit</button>
```

Comment:
@aasthashrivastava Can we change button id to submitButton? for better understanding